user/forgejo-runner/files/docker_host.env

```diff
@@ -0,0 +1 @@
+DOCKER_HOST=unix://${XDG_RUNTIME_DIR}/podman/podman.sock
```

Comment:
podman api socket is in /run/podman/podman.sock, and the service won't have a xdg_runtime_dir either, so how did this work

Reply:
Um, well, it did not